### PR TITLE
Improve help text

### LINKED ISSUE #63: Improve help text
As a new `psc-package` user I struggled yesterday to perform the most basic of tasks, an initial package set install.

Neither the help text, nor the README provide any guidance in this regard. In fact, the README seems to target package authors with little thought to package users.

With some help I discovered `psc-package update` was what I needed, but that command and the associated help text `Update dependencies` have different semantics from an initial install.

I propose improving the help text for a few of the commands:

`init`        `Initialize a new package` -> `Create a new psc-package.json file`
`update` `Update dependencies` -> `Install or update package dependencies`
`build`    `Build the current package and dependencies` -> `Update dependencies and compile the current package`

Additionally, a line near the top of the README regarding initial install would be helpful. Something along the lines of:

```sh
# installs the project dependencies listed in psc-package.json
psc-package update
```

### PR DESCRIPTION
**Pull request by gabejohnson**
Fixes #63

### CODE REVIEW

**Comment by gabejohnson** [comment]
I extended `install` to work without a specified package. I can put that in a separate PR if necessary.

File: README.md [hunk 1]
```diff
@@ -7,7 +7,23 @@
...
 
 - Download the binary for your platform from [the releases page](https://github.com/purescript/psc-package/releases), and copy it somewhere on your PATH, or
-- Build from source, using `stack install`. 
+- Build from source, using `stack install`.
+
+## Usage
+
+```shell
+# install the dependencies listed in psc-package.json
+$ psc-package install
```
Comment (paf31) [inline README.md]: Is this right?

Comment (paf31) [reply README.md]: Oh you changed it, I see.

Comment (paf31) [reply README.md]: I'd rather not have two ways to do the same thing, could you please move that to an issue or separate PR for discussion?

Comment (gabejohnson) [reply README.md]: My mistake. I mentioned it in https://github.com/purescript/psc-package/issues/63#issuecomment-336160024 and thought you had agreed. I'll submit a separate PR.

**Comment by gabejohnson** [comment]
I pulled out the changes to `install` and will submit a PR once this gets merged.

**Comment by paf31** [comment]
Thanks!

**Pull request merged**